.travis.yml

```diff
@@ -92,151 +63,149 @@ matrix:
+          env: PYTHON_VERSION=3.7
+               TOXENV="test-alldeps"
+               TOXPOSARGS="--remote-data=astropy"
+               CONDA_DEPENDENCIES="clang"
                CCOMPILER=clang
                EVENT_TYPE='cron'
-               DOUBLE_PLAY=True
+               # DOUBLE_PLAY=True
 
         # Try all python versions and Numpy versions. Since we can assume that
         # the Numpy developers have taken care of testing Numpy with different
         # versions of Python, we can vary Python and Numpy versions at the same
         # time.
-        - os: linux
-          env: PYTHON_VERSION=3.6 NUMPY_VERSION=1.16
-               INSTALL_CMD='python setup.py build_ext --inplace'
-               PIP_DEPENDENCIES='pytest-astropy'
-               TEST_CMD='pytest --open-files --doctest-rst'
-          script:
-            - $INSTALL_CMD
-            - $TEST_CMD
-
-        # Now try with all optional dependencies. We also include the --readonly
-        # flag to make sure no files are being written to the temporary install
-        # location during testing. We also use this build to make sure that the
-        # dependencies get correctly installed with pip.
-        - os: linux
-          env: SETUP_CMD='test --remote-data=astropy --readonly'
-               LC_CTYPE=C.ascii LC_ALL=C
-               PIP_DEPENDENCIES="" CONDA_DEPENDENCIES=""
-               INSTALL_WITH_PIP=True
-               EXTRAS_INSTALL="test,all"
-               PYTHON_VERSION=3.8
 
-        - os: linux
+        # For the Numpy 1.16 build we use oldestdeps not numpy116 since we can check
+        # the oldest version of all dependencies where this is known. We also check
+        # that tests do not open and leave open any files. This has a performance
+        # impact on running the tests, hence why it is not enabled by default.
+        - language: python
+          python: 3.6
+          env: TOXENV="test-oldestdeps"
+               TOXPOSARGS="--open-files"
+
+        # Now try with all optional dependencies.
+
+        - language: python
+          python: 3.6
           stage: Initial tests
-          env: PYTHON_VERSION=3.7 CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
-               PIP_DEPENDENCIES="scikit-image $ASDF_PIP_DEP"
-               SETUP_CMD='test -a "--durations=50"'
+          env: TOXENV="test-alldeps"
+               TOXPOSARGS="--durations=50""
           compiler: clang
 
         # Full tests with coverage checks.
-        - os: linux
-          env: SETUP_CMD='test --coverage --remote-data=astropy --readonly'
-               CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
-               PIP_DEPENDENCIES="scikit-image codecov objgraph jplephem bintrees sortedcontainers $ASDF_PIP_DEP"
+        - language: python
+          python: 3.7
+          env: TOXENV="test-alldeps-cov"
+               TOXPOSARGS="--remote-data=astropy"
                LC_CTYPE=C.ascii LC_ALL=C
                CFLAGS='--coverage -fno-inline-functions -O0'
```

Comment:
Is this still necessary? In `tox.ini`, it would seem this gets done automatically...

Reply:
Indeed, my bad! I added it to tox.ini without removing it here... Fixed now.